extensions/positron-python/src/client/pythonEnvironments/base/locators/lowLevel/posixKnownPathsLocator.ts

```diff
@@ -90,18 +94,13 @@ export class PosixKnownPathsLocator extends Locator<BasicEnvInfo> {
  *                    Default is 2 levels.
  * @returns Paths to Python binaries found in additional locations for Posix systems.
  */
-export async function* getAdditionalPosixDirs(searchDepth = 2): AsyncGenerator<string> {
-    const additionalLocations = [
-        // /opt/python is a recommended Python installation location on Posit Workbench.
-        // see: https://docs.posit.co/ide/server-pro/python/installing_python.html
-        '/opt/python',
-    ];
-    for (const location of additionalLocations) {
-        const additionalDirs = findInterpretersInDir(location, searchDepth);
-        for await (const dir of additionalDirs) {
-            const { filename } = dir;
+export async function* getAdditionalPosixBinaries(searchDepth = 2): AsyncGenerator<string> {
+    for (const location of ADDITIONAL_POSIX_BIN_PATHS) {
+        const executables = findInterpretersInDir(location, searchDepth);
+        for await (const entry of executables) {
+            const { filename } = entry;
             if (await looksLikeBasicGlobalPython(filename)) {
-                yield path.dirname(filename);
+                yield filename;
```

Comment:
This change fixes the duplicate python installations issue for the JS Locator. I wasn't sure where to apply a similar fix for the Native Locator. If anyone has guidance on this, happy to add in a fix as part of this PR; otherwise, I'll open an issue to address this separately.

![image](https://github.com/user-attachments/assets/6c1f101c-e9d8-488e-903e-6d9e2ebfcba3)